plugins/git4idea/src/git4idea/GitReference.java

```diff
@@ -28,7 +28,7 @@
 
   public static final TObjectHashingStrategy<String> BRANCH_NAME_HASHING_STRATEGY = FilePathHashingStrategy.create();
 
-  @NotNull protected final String myName;
+  @NotNull private final String myName;
```

Comment:
It is good to encapsulate the name, but I'd prefer it to be made in a separate commit, since it doesn't directly apply to the symbolic-ref feature.